src/python/test/help_messages/extract_expression_help_message.txt

```diff
@@ -40,7 +40,9 @@ optional arguments:
                         returned samples will be initially filtered to match
                         the cohort-defined set of samples, and any additional
                         filters will only further refine the cohort-defined
-                        set.
+                        set. If no filter is provided, an SQL query returning
```

Comment:
`If no filter is provided, an SQL ...`
to
`If no filter is provided, a SQL ...`